overlord/state/notices.go

```diff
@@ -279,15 +285,19 @@ func (s *State) AddNotice(userID *uint32, noticeType NoticeType, key string, opt
 	return notice.id, nil
 }
 
-func validateNotice(noticeType NoticeType, key string, options *AddNoticeOptions) error {
+// ValidateNotice validates notice type and key before adding.
+func ValidateNotice(noticeType NoticeType, key string, options *AddNoticeOptions) error {
 	if !noticeType.Valid() {
-		return fmt.Errorf("internal error: attempted to add notice with invalid type %q", noticeType)
+		return fmt.Errorf("attempted to add notice with invalid type %q", noticeType)
```

Comment:
I think all the "attempted to" here should be turned into "cannot"  if we remove the "internal error" prefix

Reply:
updated, thanks!